WIN32-Code/nmake/event2/event-config.h

```diff
@@ -341,24 +341,24 @@
 #define EVENT__VERSION "2.2.0-alpha-dev"
 
 /* Define to appropriate substitue if compiler doesnt have __func__ */
-#define EVENT____func__ __FUNCTION__
+#define __func__ __FUNCTION__
 
 /* Define to `__inline__' or `__inline' if that's what the C compiler
    calls it, or to nothing if 'inline' is not supported under any name.  */
-#ifndef _EVENT___cplusplus
-#define EVENT__inline __inline
+#ifndef __cplusplus
+#define inline __inline
 #endif
 
 /* Define to `int' if <sys/types.h> does not define. */
-/* #undef EVENT__pid_t */
+/* #undef pid_t */
 
 /* Define to `unsigned' if <sys/types.h> does not define. */
-/* #undef EVENT__size_t */
+/* #undef size_t */
 
 /* Define to unsigned int if you dont have it */
-#define EVENT__socklen_t unsigned int
+#define socklen_t unsigned int
 
 /* Define to `int' if <sys/types.h> does not define. */
-#define EVENT__ssize_t SSIZE_T
```

Comment:
have you tried to compile libevent with this header on win32?

Reply:
No, I don't have access to a Windows development environment.  Doesn't Travis build with AppVeyor?

Reply:
Yes it does, but not sure that it uses this pre-generated header. Personally I, would prefer to leave only `cmake` builds, this will fix a lot of caveats, but I guess that we can't do this right now.

I have some vagrant boxes (that includes windows), I hadn't used them for a few months, so it will took some time.